web_external/model.js

```diff
@@ -0,0 +1,17 @@
+isic.Model = girder.Model.extend({
+    urlRoot: function () {
+        return this.resourceName;
+    },
+
+    // "girder.Model.destroy" doesn't trigger many of the proper Backbone events
+    destroy: Backbone.Model.prototype.destroy
+});
+
+Backbone.ajax = function (options) {
```

Comment:
Do we need to override this because we're now calling Backbone's `Model.destroy` directly?